cypress/e2e/3-cypress-assertions/3-error-messages/repeate-password/ADDITIONAL-incorrect.cy.js

```diff
@@ -0,0 +1,16 @@
+describe('Check error messages for each fields', ()=>{
+    context(' page -  /auth/register', ()=>{
+
+        it('Error behind "Password "(empty input)',()=>{
+            cy.visit('/auth/register');
+            cy.get('[ng-reflect-status]').eq(2).click().type('123123')
+            cy.get('[ng-reflect-status]').eq(3).click().type('141414')
+            cy.get('[ng-reflect-status]').eq(2).click()
+            cy.get('.caption').should('contain', 'The passwords do not match')
+        })
+});
+});
+
+//  цей тест додатковий, я написала його по своїй інніціативі.
+// на данний момент, на сайті не відображається нічого, коли в 'repeate password' вводиться інший пасворд. 
+// я вирішила що це схоже на баг та вирішила дофантузувати вимогу для такого кейсу. Кейс падає бо ніякого еррор месседжу нема.
```

Comment:
yes, its a bug. But here you can verify that the border color of container is changed to red:
<img width="812" alt="Screenshot 2023-11-09 at 15 52 28" src="https://github.com/MaryIvanisovaa/cypress-test/assets/45169801/6b63b9a8-6914-40cc-b12b-a991d4895c4c">